modules/ROOT/pages/batch-filters-and-batch-aggregator.adoc

```diff
@@ -159,15 +146,20 @@ You can, for example, for example, use the Groovy scripting module to modify the
 </batch:job>
 ----
 
-The sequential access method assumes that:
+//TODO: NEED TO EXPLAIN THE FOLLOWING BULLET POINTS.
+Sequential access to records assumes the following:
 
-. The aggregator size matches the amount of aggregated records.
+. The aggregator size matches the number of aggregated records.
 . There is a direct correlation between the aggregated records and the items in the list.
 
-You can also access random records by specifying the iteration number of the foreach, saving you the need to iterate through all records. +
-The foreach scope exposes a `records` variable. This variable is an immutable list used by foreach to keep track of the iteration and provides a random access list that is accessible across the batch aggregator.
+//TODO: HOW WOULD ANYONE KNOW THE "iteration number" of the record? Is that the index of the record
```

Comment:
QUESTION: see inline TODO

//TODO: HOW WOULD ANYONE KNOW THE "iteration number" of the record? Is that the index of the record

Reply:
Added question to issue #7 in W-11934285